ARCs/arc-0033.md

```diff
@@ -0,0 +1,71 @@
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
+xGov **MAY** participate in the designation of proposals.
+
+### Requirement to become an xGov
+The Foundation suggests that to become an xGov an Algo holder **MUST**:
+- Participate in a reward-bearing AF program
+- Commit the rewards to an xGov DAO for a period
+> Each xGov DAO is related to a vertical ( art's NFT, DEFI, Community Creators ... )
+> The period of commitment tends to be a year 
+> The minimum participation to the reward-bearing AF program tends to be 1 Algo
+
+The foundation will maintain lists of eligible reward-bearing programs, and a list of qualified DAOs.
+ 
+### Duty of an xGov
+Each quarter, an xGov **MUST** vote on a certain number of proposals submitted to an xGov DAO, he will be disqualified otherwise.
+> 1 Algo commited to the rewar-bearing AF program = 1 Vote available for a proposal
+> A vote can be Yes/No/Neutral 
+> The minimum number of votes is still to be determined.
+> After a disqualification, an xGov will lose his reward committed to the DAO
+
+
+#### Example
+Alice participates in governance during Q1 2023
+the potential reward is 5 Algo, Alice:
+1) Commit 100 Algo in an AF program
+2) Express her will to participate in xGov
+3) Will not receive 5 Algo at the end of Q1 2023. They are instead locked in one big xGov pool for 1 year
+4) Will receive 5 Algo at Q1 2024
+ 
+## Rationale
+In this ARC we seek community input on issues such as, but not limited to:
+- The types of reward-bearing activities that **SHOULD** be included in the qualification list (ie. general governance)
+- Whether xGovs **MUST** vote on all measures to keep their status
+- Whether xGovs tokens **SHOULD** be sold/transferred or **SHOULD** be burned when exiting the xGov DAO
+- Whether xGovs tokens expire after a year or gain more voting power if the stake remains in the DAO
+- Whether there **SHOULD** be a minimum and a maximum amount of Algos required to become an xGov
+- Whether there be a maximum amount of Algos that can be locked by xGovs for the each period, limiting the total number of xGovs
+- Whether xGovs **SHOULD** be disqualified and lose their status if they behave in a suboptimal fashion
+- What types of suboptimal behavior would trigger disqualification or loss of status?
+- Whether new xGovs be allowed into the pool at every quarter or only once per year
+
+Community members can add their own questions and solutions to the above list
```

Comment:
Can anyone define what type of governance is Algorand Governance striving to achieve and the role of xGov in it?

Based on past discussions and G3 vote, it seems the sentiment of *1 ALGO = 1 vote* is prevalent, although it was never explicitly ratified. For future use, the term "algocracy" might be appropriate when referring to this form of governance.
But is Algorand Governance striving for a direct/pure or a representative form of "algocracy"?

Current suggestions about xGov seem to be leaning towards a representative form - i.e. with participation through different DAOs as well as due to the different requirements for Governors and xGov (which creates two classes of ALGO with different amounts of power).
With one of the main goals of blockchain being decentralization and equality, a pure form would be more suitable.
A pure form as the basis would still enable formation of representatives of different groups, which could be facilitated by the Foundation (e.g. through the recently established [Governance Advisory Council](https://www.algorand.foundation/news/improving-governance-flows-and-key-dates-for-governance-period-6)) or anyone else (e.g. formation of DAOs per vertical as currently suggested). But the Governance core would be a "pure algocracy", holding onto the same principles as in PPoS consensus.

A possible form of direct participation are initiative referendums, where groups form to bring attention to an initiative and gather initial support for it before being put to a general vote. xGovs could be the entities that start, review and endorse these initiatives, as well as be held accountable - both for the case of malicious/spam initiatives being put to the general vote and the outcomes of the passed initiatives.
A POC implementation of this approach on Algorand can be found at: https://github.com/uhudo/xGov-initiative-referendums